src/pymmcore_widgets/_mda/_stack_viewer.py

```diff
@@ -0,0 +1,284 @@
+
+        # Channels
+        nc = sequence.sizes["c"]
+        self.images = []
+        for i in range(nc):
+            image = scene.visuals.Image(
+                np.zeros(self._canvas.size).astype(self.datastore.dtype),
+                parent=self.view.scene,
+                cmap=self.cmaps[i],
+                clim=[0, 1],
+            )
+            if i > 0:
+                image.set_gl_state("additive", depth_test=False)
+            self.images.append(image)
+            self.current_channel = i
+            self._new_channel.emit(i, sequence.channels[i].config)
+
+    def _handle_channel_clim(self, values, channel: int, set_autoscale=True):
+        self.images[channel].clim = values
+        if self.channel_row.boxes[channel].autoscale_chbx.isChecked() and set_autoscale:
+            self.channel_row.boxes[channel].autoscale_chbx.setCheckState(
+                QtCore.Qt.Unchecked
+            )
+        self._canvas.update()
+
+    def _handle_channel_cmap(self, my_color: QtGui.QColor, channel: int):
+        my_color = [x // 255 for x in my_color.getRgb()[:3]]
+        self.images[channel].cmap = color.Colormap([[0, 0, 0], my_color])
+        self._canvas.update()
+
+    def _handle_channel_visibility(self, state, channel: int):
+        self.images[channel].visible = self.channel_row.boxes[
+            channel
+        ].show_channel.isChecked()
+        self._canvas.update()
+
+    def _handle_channel_autoscale(self, state, channel: int):
+        if state == 0:
+            slider = self.channel_row.boxes[channel].slider
+            self._handle_channel_clim(slider.value(), channel, set_autoscale=False)
+        else:
+            clim = (
+                np.min(self.images[channel]._data),
+                np.max(self.images[channel]._data),
+            )
+            self._handle_channel_clim(clim, channel, set_autoscale=False)
+
+    def _handle_channel_choice(self, channel: int):
+        self.current_channel = channel
+
+    def _create_sliders(self, sequence: MDASequence = None):
+        n_channels = 5 if sequence is None else sequence.sizes["c"]
+        self.channel_row = ChannelRow(n_channels, self.cmaps)
+        self.channel_row.visible.connect(self._handle_channel_visibility)
+        self.channel_row.autoscale.connect(self._handle_channel_autoscale)
+        self.channel_row.new_clims.connect(self._handle_channel_clim)
+        self.channel_row.new_cmap.connect(self._handle_channel_cmap)
+        self.channel_row.selected.connect(self._handle_channel_choice)
+        self.layout().addWidget(self.channel_row)
+
+        if sequence is not None:
+            dims = [x for x in sequence.sizes.keys() if sequence.sizes[x] > 0]
+        else:
+            dims = DIMENSIONS
+        dims.remove("c")
+        self.sliders = []
+        for dim in dims:
+            slider = LabeledVisibilitySlider(dim, orientation=QtCore.Qt.Horizontal)
+            slider.valueChanged[int, str].connect(self.on_display_timer)
```

Comment:
avoid using signal overloads ... they have variable support across different qt backends, and are phased out in qt6

Reply:
Interesting, didn't know, sorry. Will handle that when modifying the slider to use the superqt slider.